Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -469,6 +469,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
             ('2021' in key and fragment == "TTbar_14TeV"),
             ('2018' in key and fragment == "ZMM_13"),
             ('2021' in key and fragment == "ZMM_14"),
+            ('2026D87' in key and fragment == "TTbar_14TeV" and "PixelOnly" in self.suffix)
```

Comment:
@AdrianoDee D87 contains Tracker T27 which has an experimental pixel topology with a "bricked" layout, see https://github.com/cms-sw/cmssw/tree/master/Configuration/Geometry#readme. That requires a different clusterization and rechit conversion algorithm.
Please move to another geometry. I'd sugged e.g. D88 which employ T24 (i.e. regulard old standard planar rectangular pixels).

Reply:
Moved to `2026D88`